data_structures/linked_list/singly_link_list_deletion.c

```diff
@@ -70,31 +109,61 @@ void viewlist()  // function to display values
     }
 }
 //////////////////////////////////////////////////////////////////////
-
+void test(){
```

Comment:
Must be `static` for the Doxygen documentation.
```suggestion
static void test(){
```